src/main/java/org/thoughtcrime/securesms/contacts/ContactSelectionListAdapter.java

```diff
@@ -268,6 +268,8 @@ public void onBindViewHolder(@NonNull ViewHolder viewHolder, int i) {
       itemMultiSelect = false; // the item creates a new contact in the list that will be selected instead
```

Comment:
<img width="1024" height="992" alt="image" src="https://github.com/user-attachments/assets/df7b2ffa-ab2a-47c0-9629-236d5e6f2f1c" />


Can we remove the "Add Contact Manually" option from the screen that appears after clicking on `+` on the main screen? I think that 4 'new something' options are a bit overwhelming (5 with 'New Channel')

"Add Contact Manually" is available after clicking on "New E-Mail", I think this is enough.

BTW, it would anyways be nice to highlight "New Contact" a bit, because this is what most users need, but this is out of scope for this PR.

Reply:
+1, for the same reasons i removed "Add Contact Manually" from the "New Chat" on iOS, https://github.com/deltachat/deltachat-ios/pull/2770

it should continue existing below "Add recipient", "New Contact" and "Scan QR Code" (it think, it is already at all three places)